netlify.toml

```diff
@@ -20,6 +20,8 @@
       frame-src 'self' https://api-iam.intercom.io https://widget.intercom.io;
       frame-ancestors 'self';
       base-uri 'self';
+      form-action 'self';
+      navigate-to 'self';
```

Comment:
I think we should remove this. Now users cannot navigate to anywhere. They cannot go to Coinbase using our referral, they cannot go to the marketing page, ... 😅  Too much security